kubernetes-deployment/bootstrap.py

```diff
@@ -367,18 +373,39 @@ def destory_whole_cluster(cluster_config):
 
 
 
+def add_new_nodes(cluster_config, node_list_config):
+    role = node_list_config['remote_deployment_role']
+    machine_list = node_list_config['machinelist']
+    #Todo: add validation here
+    kubernetes_nodelist_deployment(cluster_config, machine_list, role, False)
+
+
+
+def remove_nodes(cluster_config, node_list_config):
+    role = node_list_config['remote_deployment_role']
+    machine_list = node_list_config['machinelist']
+    for host in machine_list:
+        execute_shell(
+            "kubectl delete node {0}".format(machine_list[host]['nodename']),
+            "Failed to delete  node {0}".format(machine_list[host]['nodename'])
+        )
+
+    # Todo: add validation here
+    kubernetes_nodelist_deployment(cluster_config, machine_list, role, True)
+
+
+
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('-d', '--deploy', action="store_true", help='Deploy kubernetes to your cluster')
     parser.add_argument('-p', '--path', required=True, help='path of cluster configuration file')
     parser.add_argument('-c', '--clean', action="store_true", help="clean the generated script")
+    parser.add_argument('-f', '--file', default=None, help="An yamlfile with the nodelist to maintain")
+    parser.add_argument('-a', '--add', action="store_true", help="Add the node from nodelist.yaml")
+    parser.add_argument('-r', '--remove', action="store_true", help="Remove the node from nodelist.yaml")
```

Comment:
Is it easy to add a repair command that if a machine is bad, we can use repair command to reinstall Kuberlets on it.

Reply:
we will make the option only repair one node or a nodelist? @sterowang